Library/Homebrew/test/formulary_spec.rb

```diff
@@ -135,7 +135,14 @@ class Wrong#{described_class.class_s(formula_name)} < Formula
       end
     end
 
-    context "with installed Formula" do
+    context "with installed Formula", :integration_test do
+      before do
```

Comment:
Given this identical block is used in a few places a helper method in `spec_helper` would probably be preferable?

Reply:
There's now two places that use two identical lines.
```ruby
        stub_formula_loader formula("patchelf") { url "patchelf-1.0" }
        allow(Formula["patchelf"]).to receive(:installed?).and_return(true)
```
I haven't factored these two lines out into their own helper method, but I could if you like.